EIPS/eip-5732.md

```diff
@@ -151,6 +154,33 @@ contract BarVote is ISimpleVote, IERC_COMMIT {
     // Proposal deadline method omitted.
 }
 ```
+
+### Commit with ENS Register as Reveal
+
+In ENS registering process currenctly
+incide of ETHRegistrarController contract
+a commit function is being used to
+allow registerers fairly register a desire
+domain to avoid being frontrunned, it looks loke this
```

Comment:
I'm very confused. Would you mind fixing this?

Reply:
Good catch. I fixed it.